### PR TITLE
[NXCM-3666] Set log level of BasicHttpAuthenticationFilter to INFO

### PR DESCRIPTION
**Pull request by adreghiciu**
Signed-off-by: Alin Dreghiciu adreghiciu@gmail.com

### CODE REVIEW

**Comment by nabcos** [comment]
+1

File: nexus/nexus-logging-extras/src/main/resources/META-INF/log/logback-nexus.xml [hunk 1]
```diff
@@ -38,12 +38,13 @@
...
   
   <logger name="org.jsecurity" level="WARN" />
-  <logger name="httpclient" level="INFO" />
+  <logger name="org.apache.commons" level="WARN" />
   <logger name="org.sonatype.nexus.rest.NexusApplication" level="ERROR" />
   <logger name="org.jsecurity.subject.AbstractRememberMeManager" level="ERROR" />
   <logger name="org.apache.http" level="INFO" />
   <logger name="org.restlet" level="WARN" />
   <logger name="org.apache.commons" level="WARN" />
+  <logger name="org.apache.shiro.web.filter.authc.BasicHttpAuthenticationFilter" level="WARN" />
```
Comment (velo) [inline nexus/nexus-logging-extras/src/main/resources/META-INF/log/logback-nexus.xml]: The description say "Set log level of BasicHttpAuthenticationFilter to INFO" but you set level to WARN....

Comment (adreghiciu) [inline nexus/nexus-logging-extras/src/main/resources/META-INF/log/logback-nexus.xml]: Fixed by next commit

**Comment by mcculls** [comment]
+1

File: nexus/nexus-logging-extras/src/main/resources/META-INF/log/logback-nexus.xml [hunk 2]
```diff
@@ -38,12 +38,13 @@
   </appender>
   
   <logger name="org.jsecurity" level="WARN" />
-  <logger name="httpclient" level="INFO" />
+  <logger name="org.apache.commons" level="WARN" />
   <logger name="org.sonatype.nexus.rest.NexusApplication" level="ERROR" />
```
Comment (peterlynch) [inline nexus/nexus-logging-extras/src/main/resources/META-INF/log/logback-nexus.xml:42]: Why the above two changes? They seem unrelated. What impact will these have?

Comment (adreghiciu) [inline nexus/nexus-logging-extras/src/main/resources/META-INF/log/logback-nexus.xml:42]: I do not know how they went in . I will close and remake it.

**Comment by adreghiciu** [comment]
Closing issue as unwanted changes got into it
